Makefile

```diff
@@ -315,9 +311,35 @@ dev-deploy: deploy-cluster dev-infra dev-dracon
 dev-teardown:
 	@kind delete clusters dracon-demo
 
-generate-protos: install-lint-tools
-	@echo "Generating Protos"
-	@buf generate
+build_buf_container:
```

Comment:
Tiny code style one: all other `make` commands use kebab case. Could you change that to match them please?